src/command.js

```diff
@@ -221,4 +228,31 @@ export default class StorybookCommand extends Command {
       await page?.close();
     }
   }
+
+  async getStorybookVersion(url) {
+    let version, page;
+
+    try {
+      page = await this.percy.browser.page();
+      this.log.debug(`Get Storybook version: ${url}/?path=/settings/about`);
+
+      await page.goto(`${url}/?path=/settings/about`);
+      /* istanbul ignore next: no instrumenting injected code */
+      version = await page.eval(async ({ waitFor }) => {
+        await waitFor(() => !!document.querySelector('header'), 5000)
+          .catch(() => Promise.reject(new Error('Failed to find a <header> element')));
+        let getPath = path => document.evaluate(path, document, null, 9, null).singleNodeValue;
```

Comment:
Could move `getPath` to before the wait, then utilize it when waiting so we wait on the exact element we're looking for.